mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceStatusDTO.java

```diff
@@ -40,38 +45,147 @@
+        properties = {
+            @ExtensionProperty(name = "title", value = "RequestedReplicas"),
+            @ExtensionProperty(name = "x-order", value = "40"),
+            @ExtensionProperty(name = "description", value = "Counter for number of replicas of a MicoService that should be available.")
+        }
+    )})
     private int requestedReplicas;
 
     /**
-     * Counter for that are actually available.
+     * Counter for replicas that are actually available.
       */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "AvailableReplicas"),
+            @ExtensionProperty(name = "x-order", value = "50"),
+            @ExtensionProperty(name = "description", value = "Counter for replicas of a MicoService that are actually available.")
+        }
+    )})
     private int availableReplicas;
 
     /**
      * Each item in this list represents a Kubernetes Service.
      */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "InterfacesInformation"),
+            @ExtensionProperty(name = "x-order", value = "60"),
+            @ExtensionProperty(name = "description", value = "Each item in this list represents a Kubernetes Service.")
+        }
+    )})
     private List<MicoServiceInterfaceDTO> interfacesInformation = new ArrayList<>();
 
     /**
-     * List of {@link io.fabric8.kubernetes.api.model.Pod}s of a {@link io.github.ust.mico.core.model.MicoService}.
+     * List of {@link MicoUsingApplicationDTO}, representing all applications that are using one service together.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "UsingApplications"),
+            @ExtensionProperty(name = "x-order", value = "70"),
+            @ExtensionProperty(name = "description", value = "List of MicoUsingApplicationDTO, representing all applications that are using one service together.")
+        }
+    )})
+    private List<MicoUsingApplicationDTO> usingApplications = new ArrayList<>();
+
+    /**
+     * List of all {@link Pod}s of all replicas of a deployment of a {@link MicoService}.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "PodsInformation"),
+            @ExtensionProperty(name = "x-order", value = "80"),
+            @ExtensionProperty(name = "description", value = "List of all pods of all replicas of a deployment of a MicoService.")
+        }
+    )})
+    private List<KubernetesPodInfoDTO> podsInformation = new ArrayList<>();
```

Comment:
`podsInfo`would be better, and conform to the class name.

Reply:
Change **KubernetesPodInfoDTO** to **KubernetesPodInformationDTO**